js/tests/components/SEMrushCountrySelector.test.js

```diff
@@ -0,0 +1,34 @@
+import { shallow } from "enzyme";
+
+import SEMrushCountrySelector
+	from "../../../js/src/components/modals/SEMrushCountrySelector";
+import React from "react";
+import { noop } from "lodash";
+
+window.jQuery = () => ( {
+	select2: () => {
+	}, on: () => {
+	},
+} );
```

Comment:
Indentation here is a bit confusing, will push a small change but it's OK anyway

Reply:
Done